packages/react-components/react-textarea/src/stories/Textarea.stories.tsx

```diff
@@ -5,12 +5,12 @@ import bestPracticesMd from './TextareaBestPractices.md';
 
 export { Default } from './TextareaDefault.stories';
 export { Appearance } from './TextareaAppearance.stories';
-export { Controlled } from './TextareaControlled.stories';
 export { Disabled } from './TextareaDisabled.stories';
 export { Placeholder } from './TextareaPlaceholder.stories';
 export { Resize } from './TextareaResize.stories';
 export { Size } from './TextareaSize.stories';
 export { Uncontrolled } from './TextareaUncontrolled.stories';
+export { Controlled } from './TextareaControlled.stories';
```

Comment:
Order of exports matters. I changed it to have controlled & uncontrolled examples next to each other.